python/py-halo/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-halo
+version             0.0.31
+revision            0
+
+license             MIT
+supported_archs     noarch
+platforms           {darwin any}
+
+maintainers         {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description         Beautiful terminal spinners in Python
+long_description    {*}${description}
+
+homepage            https://github.com/manrajgrover/halo
+
+distname            halo-${version}
```

Comment:
is this required - I *think* it's the default already?

Reply:
fixed